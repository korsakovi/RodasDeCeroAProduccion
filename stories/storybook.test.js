// storybook.test.js

/* Object.defineProperty(window, 'matchMedia', {
	writable: true,
	value: jest.fn().mockImplementation((query) => ({
		matches: false,
		media: query,
		onchange: null,
		addListener: jest.fn(), // deprecated
		removeListener: jest.fn(), // deprecated
		addEventListener: jest.fn(),
		removeEventListener: jest.fn(),
		dispatchEvent: jest.fn(),
	})),
}); */

import initStoryshots from '@storybook/addon-storyshots';

initStoryshots();

//import React from 'react';
/* import Button from '../atoms/Button';
import rendered from 'react-test-renderer';

describe('Button component', () => {
	it('should match the snapshot', () => {
		const component = rendered.create(<Button>Button</Button>);
		let tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
}); */
