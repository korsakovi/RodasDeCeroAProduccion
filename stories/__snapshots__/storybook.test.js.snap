// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Atoms/Button Default 1`] = `
<button
  className="button is-block"
  onClick={[Function]}
>
  Default Button
</button>
`;

exports[`Storyshots Atoms/Button Primary 1`] = `
<button
  className="button type-primary is-block"
  onClick={[Function]}
>
  Primary Button
</button>
`;

exports[`Storyshots Atoms/Button Secondary 1`] = `
<button
  className="button type-secondary is-block"
  onClick={[Function]}
>
  Secondary Button
</button>
`;

exports[`Storyshots Atoms/Button Tertiary 1`] = `
<button
  className="button type-tertiary"
  onClick={[Function]}
>
  Tertiary Button
</button>
`;

exports[`Storyshots Atoms/Heading Colors 1`] = `
<div
  style={
    {
      "display": "flex",
      "flexDirection": "column",
    }
  }
>
  <header
    className="heading color-default size-md"
  >
    Default Color
  </header>
  <header
    className="heading color-primary size-md"
  >
    Primary Color
  </header>
</div>
`;

exports[`Storyshots Atoms/Heading Default 1`] = `
<div
  style={
    {
      "display": "flex",
      "flexDirection": "column",
    }
  }
>
  <header
    className="heading color-default size-md"
  >
    Default Heading
  </header>
</div>
`;

exports[`Storyshots Atoms/Heading Sizes 1`] = `
<div
  style={
    {
      "display": "flex",
      "flexDirection": "column",
    }
  }
>
  <header
    className="heading color-default size-xs"
  >
    El veloz murciélago hindú comía feliz cardillo y kiwi.
  </header>
  <header
    className="heading color-default size-md"
  >
    El veloz murciélago hindú comía feliz cardillo y kiwi.
  </header>
  <header
    className="heading color-default size-lg"
  >
    El veloz murciélago hindú comía feliz cardillo y kiwi.
  </header>
  <header
    className="heading color-default size-xl"
  >
    El veloz murciélago hindú comía feliz cardillo y kiwi.
  </header>
  <header
    className="heading color-default size-2xl"
  >
    El veloz murciélago hindú comía feliz cardillo y kiwi.
  </header>
</div>
`;

exports[`Storyshots Atoms/Icon Angle Down 1`] = `
<div
  className="icon"
  style={
    {
      "height": 26,
      "width": 26,
    }
  }
>
  <picture
    className="picture"
  >
    <img
      src="/icons//angle-down.svg"
      style={
        {
          "maxWidth": "26px",
        }
      }
    />
  </picture>
</div>
`;

exports[`Storyshots Atoms/Icon Arrow Right 1`] = `
<div
  className="icon"
  style={
    {
      "height": 26,
      "width": 26,
    }
  }
>
  <picture
    className="picture"
  >
    <img
      src="/icons//arrow-right.svg"
      style={
        {
          "maxWidth": "26px",
        }
      }
    />
  </picture>
</div>
`;

exports[`Storyshots Atoms/Icon Has Background 1`] = `
<div
  className="icon has-background"
  style={
    {
      "height": 26,
      "width": 26,
    }
  }
>
  <picture
    className="picture"
  >
    <img
      src="/icons//angle-down.svg"
      style={
        {
          "maxWidth": "26px",
        }
      }
    />
  </picture>
</div>
`;

exports[`Storyshots Atoms/Input Text 1`] = `
<input
  className="input"
  onChange={[Function]}
  placeholder="Nombres"
  type="text"
  value=""
/>
`;

exports[`Storyshots Atoms/Picture Default 1`] = `
<picture
  className="picture"
>
  <img
    src="https://placeimg.com/400/400/any"
    style={
      {
        "maxWidth": "200px",
      }
    }
  />
</picture>
`;

exports[`Storyshots Layout/Spacer/Default Default 1`] = `
<div
  className="spacer is-visible"
  style={
    {
      "display": "inline-block",
      "height": undefined,
      "width": undefined,
    }
  }
/>
`;

exports[`Storyshots Layout/Spacer/Default Extra Small 1`] = `
<div
  className="spacer is-visible"
  style={
    {
      "display": "inline-block",
      "height": 5,
      "width": 5,
    }
  }
/>
`;

exports[`Storyshots Layout/Spacer/Default Large 1`] = `
<div
  className="spacer is-visible"
  style={
    {
      "display": "inline-block",
      "height": 40,
      "width": 40,
    }
  }
/>
`;

exports[`Storyshots Layout/Spacer/Default Medium 1`] = `
<div
  className="spacer is-visible"
  style={
    {
      "display": "inline-block",
      "height": 20,
      "width": 20,
    }
  }
/>
`;

exports[`Storyshots Layout/Spacer/Default Small 1`] = `
<div
  className="spacer is-visible"
  style={
    {
      "display": "inline-block",
      "height": 15,
      "width": 15,
    }
  }
/>
`;

exports[`Storyshots Layout/Spacer/Horizontal Default 1`] = `
<div
  className="spacer is-visible"
  style={
    {
      "display": "block",
      "height": undefined,
      "width": "100%",
    }
  }
/>
`;

exports[`Storyshots Layout/Spacer/Horizontal Extra Small 1`] = `
<div
  className="spacer is-visible"
  style={
    {
      "display": "block",
      "height": 5,
      "width": "100%",
    }
  }
/>
`;

exports[`Storyshots Layout/Spacer/Horizontal Large 1`] = `
<div
  className="spacer is-visible"
  style={
    {
      "display": "block",
      "height": 40,
      "width": "100%",
    }
  }
/>
`;

exports[`Storyshots Layout/Spacer/Horizontal Medium 1`] = `
<div
  className="spacer is-visible"
  style={
    {
      "display": "block",
      "height": 20,
      "width": "100%",
    }
  }
/>
`;

exports[`Storyshots Layout/Spacer/Horizontal Small 1`] = `
<div
  className="spacer is-visible"
  style={
    {
      "display": "block",
      "height": 15,
      "width": "100%",
    }
  }
/>
`;

exports[`Storyshots Layout/Spacer/Vertical Default 1`] = `
<div
  className="spacer is-visible"
  style={
    {
      "display": "block",
      "height": "100vh",
      "maxHeight": 200,
      "width": undefined,
    }
  }
/>
`;

exports[`Storyshots Layout/Spacer/Vertical Extra Small 1`] = `
<div
  className="spacer is-visible"
  style={
    {
      "display": "block",
      "height": "100vh",
      "maxHeight": 200,
      "width": 5,
    }
  }
/>
`;

exports[`Storyshots Layout/Spacer/Vertical Large 1`] = `
<div
  className="spacer is-visible"
  style={
    {
      "display": "block",
      "height": "100vh",
      "maxHeight": 200,
      "width": 40,
    }
  }
/>
`;

exports[`Storyshots Layout/Spacer/Vertical Medium 1`] = `
<div
  className="spacer is-visible"
  style={
    {
      "display": "block",
      "height": "100vh",
      "maxHeight": 200,
      "width": 20,
    }
  }
/>
`;

exports[`Storyshots Layout/Spacer/Vertical Small 1`] = `
<div
  className="spacer is-visible"
  style={
    {
      "display": "block",
      "height": "100vh",
      "maxHeight": 200,
      "width": 15,
    }
  }
/>
`;

exports[`Storyshots Molecules/ButtonIcon Default 1`] = `
<button
  className="button type-secondary"
  onClick={[Function]}
>
  <span
    style={
      {
        "marginRight": 10,
      }
    }
  >
    Button icon
  </span>
  <div
    className="icon"
    style={
      {
        "height": 26,
        "width": 26,
      }
    }
  >
    <picture
      className="picture"
    >
      <img
        src="/icons//undefined.svg"
        style={
          {
            "maxWidth": "26px",
          }
        }
      />
    </picture>
  </div>
</button>
`;

exports[`Storyshots Molecules/Dropdown Default 1`] = `
<div
  className="dropdown"
>
  <div
    className="icon has-background"
    style={
      {
        "height": 26,
        "width": 26,
      }
    }
  >
    <picture
      className="picture"
    >
      <img
        src="/icons//angle-down.svg"
        style={
          {
            "maxWidth": "26px",
          }
        }
      />
    </picture>
  </div>
  <select
    className="dropdown-select"
    onChange={[Function]}
    value=""
  >
    <option
      value={800}
    >
      8:00 AM
    </option>
    <option
      value={1000}
    >
      10:00 AM
    </option>
    <option
      value={1300}
    >
      1:00 PM
    </option>
    <option
      value={1500}
    >
      3:00 PM
    </option>
  </select>
</div>
`;
